src/chromium/cargo.md

```diff
@@ -0,0 +1,29 @@
+# Using cargo for tools
+
+As you'll see, Chromium builds don't use `cargo`. But typical Rust development
+uses `cargo` and requires downloading and running content from the
+internet:
+
+* The Rust toolchain (`cargo` and `rustc` at least)
+* Whatever [third-party crates][0] your project depends upon. (These may
+  include procedural macros and build scripts which are run on your machine
+  during the build process, as well as at run time.)
```

Comment:
Perceived problem: The "as well as at run time" part seems confusing, because proc macros and build scripts only run at build time.

Maybe this tries to say that running (non-proc-macro) third-party crate code is risky.

Reply:
Yeah, I think the parenthesized bit isn't necessary, I've removed it.